detections/endpoint/detect_mimikatz_with_powershell_script_block_logging.yml

```diff
@@ -0,0 +1,56 @@
+name: Detect Mimikatz With PowerShell Script Block Logging
+id: 8148c29c-c952-11eb-9255-acde48001122
+version: 1
+date: '2021-06-09'
+author: Michael Haag, Splunk
+type: batch
+datamodel: []
+description: 'The following analytic utilizes PowerShell Script Block Logging (EventCode=4104)
+  to identify suspicious PowerShell execution. Script Block Logging captures the command
+  sent to PowerShell, the full command to be executed. Upon enabling, logs will output
+  to Windows event logs. Dependent upon volume, enable no critical endpoints or all.
+  \
+
+  This analytic identifies common Mimikatz functions that may be identified in the
+  script block, including `mimikatz`. This will catch the most basic use cases for
+  Pass the Ticket, Pass the Hash and `-DumprCreds`. \
+
+  During triage, review parallel processes using an EDR product or 4688 events. It
+  will be important to understand the timeline of events around this activity. Review
+  the entire logged PowerShell script block.'
+search: '`powershell` EventCode=4104 Message IN (*mimikatz*, *-dumpcr*, *sekurlsa::pth*,
```

Comment:
@MHaggis I did some of this for SSA - some of the things we could add here are common calls when someone just breaks in: 
- token::whoami
- privilege::debug (trying to become an admin)
- crypto::capi
- kerberos::list
- crypto::keys